<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(195, 195, 195);

QLabel{
	Layout:center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="SideBar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>650</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(106, 106, 106);
	border: 1px;
}






</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>260</width>
       <height>766</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QPushButton" name="CV_Button">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(238, 238, 238);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;
text-align: center;

}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width: 2px;
	border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Categorize Voice</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="CV_List">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="TV_Button">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(238, 238, 238);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;
}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width: 2px;
	border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Train Voice</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Tenor_Bttn">
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(195, 195, 195);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 2px;
}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width:1px;
	border-radius: 2px;
}</string>
          </property>
          <property name="text">
           <string>Tenor</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="Tenor_List_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Soprano_Bttn">
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(195, 195, 195);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 2px;
}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width:1px;
	border-radius: 2px;
}</string>
          </property>
          <property name="text">
           <string>Soprano</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="Soprano_List_2">
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Bass_Bttn">
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(195, 195, 195);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 2px;
}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width:1px;
	border-radius: 2px;
}</string>
          </property>
          <property name="text">
           <string>Bass</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="Bass_List_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Alto_Bttn">
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(195, 195, 195);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 2px;
}

QPushButton:checked{
	color:rgb(255, 255, 255);
	font-weight:bold;
	background-color: rgb(39, 39, 39);
	border-style: solid;
	border-color: white;
	border-width:1px;
	border-radius: 2px;
}</string>
          </property>
          <property name="text">
           <string>Alto</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="Alto_List_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>150</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>261</width>
            <height>111</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>157</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Vocal_Icon">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Pictures/Raw_Image/MainLogo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Vocal_Name">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgb(226, 226, 226);}</string>
        </property>
        <property name="text">
         <string>VOCALIZER</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="MainBar" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>761</width>
      <height>651</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(195, 195, 195);
	border:1px;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>251</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>251</width>
       <height>231</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>251</width>
       <height>231</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Pictures/Raw_Image/MainLogo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Pause_Start_B">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>540</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/Pictures/Raw_Image/Pause_bttn.png</normaloff>:/Pictures/Raw_Image/Pause_bttn.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>540</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/Pictures/Raw_Image/Stop_bttn.png</normaloff>:/Pictures/Raw_Image/Stop_bttn.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="display_record">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="markdown">
      <string>Vocalizer

</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Vocalizer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
     <property name="cursorWidth">
      <number>1</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Tenor_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Tenor_List_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Tenor_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Tenor_List_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Soprano_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Soprano_List_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Soprano_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Soprano_List_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bass_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Bass_List_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bass_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Bass_List_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Alto_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Alto_List_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Alto_Bttn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Alto_List_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
